templates/nexus.properties.erb

```diff
@@ -0,0 +1,31 @@
+# Jetty section
+<% if node['nexus_repository_manager']['nexus_properties']['application_port'] != '8081' %>
```

Comment:
i don't understand this at all. the else branch does not seem to have any effect. what is the intention of this ?

Reply:
As discussed in the standup this morning, this is uncommenting this line only if the port is not 8081.

The idea is to leave the configuration settings as they are installed normally, only changing them when configured to values other than the default.

Reply:
Seems a lot more confusing this way. What if the default port changes from 8081 but we don't update our implementation? Might as well uncomment regardless since we are providing a method of configuring this value.

Reply:
Uncommented the commented configuration lines and used the configuration values.

The exception here is the nexus.licenseFile property. If this is not set, it must be commented out; otherwise, an exception is seen in the log (d9377a1).